fv3core/stencils/c_sw.py

```diff
@@ -252,7 +292,38 @@ def compute(delp, pt, u, v, w, uc, vc, ua, va, ut, vt, divgd, omga, dt2):
         origin=geo_origin,
         domain=(grid.nic + 2, grid.njc + 2, grid.npz),
     )
-    ke, vort = ke_c_sw.compute(uc, vc, u, v, ua, va, dt2)
+
+    # ke_c_sw
+    # {
+    # Create storage objects to hold the new vorticity and kinetic energy values
+    ke = utils.make_storage_from_shape(uc.shape)
+    vort = utils.make_storage_from_shape(vc.shape)
+
+    # Set vorticity and kinetic energy values
+    update_vorticity_and_kinetic_energy(
+        ke,
+        vort,
+        ua,
+        va,
+        uc,
+        vc,
+        u,
+        v,
+        grid.sin_sg1,
+        grid.cos_sg1,
+        grid.sin_sg2,
+        grid.cos_sg2,
+        grid.sin_sg3,
+        grid.cos_sg3,
+        grid.sin_sg4,
+        grid.cos_sg4,
+        dt2,
+        origin=(grid.is_ - 1, grid.js - 1, 0),
+        domain=(grid.nic + 2, grid.njc + 2, grid.npz),
+    )
+    # }
+
+    # ke, vort = ke_c_sw.compute(uc, vc, u, v, ua, va, dt2)
```

Comment:
should we delete this since it is no longer used?

Reply:
Yeah, I'll remove this in a follow-up PR, as I'm still working in `c_sw.py` for the next few in this series.